spec/factories/comments.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :comment do
+    body { "MyString" }
+    product_id { 1 }
```

Comment:
Please, not use hard code. If you need to use belongs_to record create it
product_id { create[:product].id }